Doc/library/profile.rst

```diff
@@ -121,6 +121,14 @@ results to a file by specifying a filename to the :func:`run` function::
 The :class:`pstats.Stats` class reads profile results from a file and formats
 them in various ways.
 
+You can also do inline profiling with :class:`cProfile.Profile`::
```

Comment:
Hmm, there is still a broken link.

On another hand, this example seems to be redundant: there is such one, right in the description of the Profile class.

Reply:
The reason this example is added is that I believe this example should be in the "instant manual" - which the users who are not interested in the details of cProfile/profile can refer to when they just want a quick profile. I'd say, in many (if not most) cases, users do not want to convert their code into a string - so this kind of inline profiling is what they need.

Yes there's the broken link, but that's how the other part of the docs use it ... The problem is this document is technically ``profile``, but we also put ``cProfile`` stuff in it. The context usage is only available for `cProfile` so `cProfile` should be explicit. We do either this, like the other part of the documentation, or a raw code.

Reply:
Yes this works. Nice trick!